### PR TITLE
Add AI Collection

### PR DESCRIPTION
**Pull request by kerbelp**
https://github.com/ai-collection/ai-collection

It is a collection of awesome generative AI applications that have place for more visibility on the web. Every website is checked for quality and links are periodically checked using a Lychee GitHub Action. Websites that are not responsive are removed from the list. 
The list passes the awesome-linter and I will make sure that it stays this way.
The list does not starts with an `awesome-` prefix but I saw awesome lists that does not start with it so trying to leave the name as is (e.g https://github.com/papers-we-love/papers-we-love)

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -897,6 +897,7 @@
 - [Computational Geometry](https://github.com/atkirtland/awesome-computational-geometry#readme) - Computational approaches for problems in geometry.
 - [OpenStreetMap](https://github.com/osmlab/awesome-openstreetmap#readme) - An open data mapping project utilized by many apps and devices.
 - [Computational Biology](https://github.com/inoue0426/awesome-computational-biology#readme) - Computational approaches applied to problems in biology.
+- [AI Collection](https://github.com/ai-collection/ai-collection) - A Collection of Awesome Generative AI Applications
```
Comment (hallvaaw) [inline readme.md:900]: The link should include readme like this: "https://github.com/ai-collection/ai-collection#readme".
Also, the entry must end with a period and not a trailing whitespace.

**Comment by hallvaaw** [comment]
Hi @kerbelp,

Have you reviewed two other pull requests in this repository? If so, could you list them?

File: readme.md (hunk 1 above)
Comment (hallvaaw) [reply readme.md:900]: You should also change the license to something else such as CC0 rather than the MIT license as this one is not accepted.

Comment (LeoDog896) [inline readme.md:900]: You have CI that reports for markdown mistakes in accordance with lychee, but not with awesome-lint. As an extra measure, you should add that as a PR check.

Comment (kerbelp) [reply readme.md]: Lychee is used to validate that links are not broken. I will add awesome lint as well to verify no duplicated links.

**Comment by kerbelp** [comment]
> 

I reviewed:
https://github.com/sindresorhus/awesome/pull/2511
https://github.com/sindresorhus/awesome/pull/2513

**Comment by kerbelp** [comment]
unicorn

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by kerbelp** [comment]
@sindresorhus from my understanding, the two sections that my list does not follow are:
1. My list name does not start with a prefix of Awesome
2. The structure of the list is a table and not Title - Description.

I believe that my list will bring more value in a table structure, as it adds more information about the different items (Offer Free Version).

I made sure that there are no linter issues, so please let me know if these are strict rules and if so, I will close these PR.
Thanks.

**Comment by praveen-klaytn** [comment]
Hi @kerbelp ,  
  
Thanks for wonderful AI collection list.  
  
Looks like few characters from [https://github.com/ai-collection/ai-collection/blob/main/README.md](https://github.com/ai-collection/ai-collection/blob/main/README.md) are not shown properly. its showing �� as mentioned below. few of them are listed down. but there are few more such characters in the readme file. It would be nice if those are handled.

*    Weâ��ve helped over 100,000 people land their dream jobs
*   Powered by AI it answers your teamâ��s questions inside Slack
*   Oasis Â�� Never Worry How You Look

**Comment by kerbelp** [comment]
Hi @praveen-klaytn
Thanks for the comment.
I fixed the bad characters in ai-collection 🙏

**Review by praveen-klaytn — APPROVED** [review]

**Comment by sindresorhus** [comment]
Seems like the list should be called `Awesome AI Apps` since it's a about apps. "collection" is a too generic term and could be anything.

**Comment by sindresorhus** [comment]
It also feels like most of the "Title" and "Description"column content is simply copy-pasted from the website without any care to the quality.

**Comment by sindresorhus** [comment]
Feels like too much content. I would try to reduce the content and increase the quality (including the quality of how the entries are described).

**Comment by kerbelp** [comment]
Thanks for the input @sindresorhus, I have a plan to increase quality and reduce content.
Will ping here once its ready.

**Comment by relatedcode** [comment]
Hi,
Thanks for making this awesome list.
Some feedbacks:

- There is too much content in the list and the readers might not be able to search correctly within it.
- The logo should be "either centered, fullwidth, or placed at the top-right of the readme" according to the [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md).

Thanks!

**Comment by relatedcode** [comment]
Hi,
Thanks for making this awesome list.
Some feedbacks:

- There is too much content in the list and the readers might not be able to search correctly within it.
- The logo should be "either centered, fullwidth, or placed at the top-right of the readme" according to the [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md).

Thanks!

**Comment by relatedcode** [comment]
Hi,
Thanks for making this awesome list.
Some feedbacks:

- There is too much content in the list and the readers might not be able to search correctly within it.
- The logo should be "either centered, fullwidth, or placed at the top-right of the readme" according to the [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md).

Thanks!

**Comment by relatedcode** [comment]
Hi,
Thanks for making this awesome list.
Some feedbacks:

- There is too much content in the list and the readers might not be able to search correctly within it.
- The logo should be "either centered, fullwidth, or placed at the top-right of the readme" according to the [guidelines](https://github.com/sindresorhus/awesome/blob/main/pull_request_template.md).

Thanks!

**Comment by sindresorhus** [comment]
Bump

**Pull request closed without merging**